### ISSUE TITLE
Clicking Submit on Generic Contact Us form appears to do nothing

### ISSUE THREAD

**Issue by damonlynch** [canonical-web-and-design/canonical.com#281]
Hi,

I filled out the [contact us form](https://ubuntu.com/contact-us/form?product=generic-contact-us). It asked me to fill in my company name (which is surprising considering not everyone works for a company). I put in something generic, clicked submit several times, and nothing happens. I have no idea if my message has been sent or not. 

Damon

**Comment by SirSamTumless** [comment]
Hi Damon - just to confirm it didn't redirect you to https://ubuntu.com/contact-us/form/thank-you

**Comment by damonlynch** [comment]
Yes that's correct. I never saw that page. I do wonder if it was because of me not having filled in all the required fields when I first clicked submit? To be clear, I did fill them in, but then clicking submit did nothing.

**Comment by pmahnke** [comment]
@damonlynch really sorry it didn't work.  I checked and it didn't go through to the backend either.

Can I ask if you have javascript turned off and what browser are you using? In the meantime, do you want to email sales@canonical.com with your query?

**Comment by damonlynch** [comment]
@pmahnke no worries. Thanks for checking the backend.

I don't have javascript disabled or anything like that. I am using Firefox as packaged in Ubuntu 20.04, i.e. the regular .deb package. Since my message didn't work last time, I tried the form again, and it went through with no issue. 

My memory is hazy, but I don't know if I saw the "I'm not a robot" challenge last time. So perhaps my issue was related to a temporary problem with that? It's difficult to say. I wish I could be more helpful.

Given it worked this time. perhaps it's okay to close the issue?
